packages/manager/modules/pci/src/components/project/users/users.html

```diff
@@ -133,6 +133,16 @@
                     data-translate="pci_projects_project_users_modify_password_label"
                 ></span>
             </oui-action-menu-item>
+
+            <oui-action-menu-item
+                data-on-click="$ctrl.generateCredentials($row)"
+                data-disabled="$ctrl.isActionDisabled()"
```

Comment:
i think name is to generic, may be we can rename it.
```suggestion
                data-disabled="$ctrl.canGenerateCredentials()"
```

Reply:
Hi @mohammed-zahaf , this is reused for all actions based on user status of account. Hence its not been changed. I am reusing the existing generic function here.

Reply:
Ok got it